src/Lykke.SettingsReader/SettingsProcessor/CheckFieldResult.cs

```diff
@@ -32,6 +33,7 @@ public static CheckFieldResult Ok(string propertyName, string url)
                 Url = url,
                 Result = true,
                 PropertyName = propertyName,
+                _error = null
```

Comment:
_error field will be null by default. It's a bad practice to init private fields like this.